packages/lesswrong/server/languageModels/promptUtils.ts

```diff
@@ -146,36 +147,36 @@ Title: ${post.title}
 Author: ${authorName}
 Publish Date: ${post.postedAt}
 Score: ${post.baseScore}
-Content: ${markdown}`;
+Content: ${markdown?.slice(0, truncation)}`;
 }
 
-const formatAdditionalPostsForPrompt = (posts: PostsPage[], tokenCounter: TokenCounter): string => {
-  const formattedPosts = posts.map(post => formatPostForPrompt(post));
+const formatAdditionalPostsForPrompt = (posts: LlmPost[], tokenCounter: TokenCounter, limit=120_000, prefix="Supplementary Post", truncation?: number): string => {
```

Comment:
Having the `truncation` arg refer to characters(?) here and tokens(?) in `formatPostForPrompt` is a bit confusing; I'd give them different names

Reply:
Good point